code/modules/urist/items/clothes/uristclothes.dm

```diff
@@ -940,6 +940,127 @@ Update 26/07/2014 - All generic clothing goes under obj/item/clothing/under/uris
+	item_state = "Lieut_General_US"
+
+/obj/item/clothing/under/urist/USBrigGen
+	name = "US Army Brigadier General Uniform"
+	desc = "A uniform worn by an army commander."
+	icon_state = "Brig General US"
+	item_state = "Brig General US"
+
+/obj/item/clothing/under/urist/USCol
+	name = "US Army Colonel Uniform"
+	desc = "A uniform worn by an infantry commander."
+	icon_state = "Colonel of Infantry US"
+	item_state = "Colonel of Infantry US"
+
+/obj/item/clothing/under/urist/USPrivate
+	name = "US Army Private Uniform"
+	desc = "A uniform worn by an infantryman."
+	icon_state = "Private Infantry US"
+	item_state = "Private Infantry US"
+
+/obj/item/clothing/under/urist/CSAPrivate
+	name = "CS Army Private Uniform"
+	desc = "A uniform worn by an infantryman."
+	icon_state = "Private Infantry CS"
+	item_state = "Private Infantry CS"
+
+/obj/item/clothing/under/urist/CSALt
+	name = "CS Army Lieutenant Uniform"
+	desc = "A uniform worn by an infantry commander."
+	icon_state = "Lieutenant CS"
+	item_state = "Lieutenant CS"
+
+/obj/item/clothing/under/urist/CSAArty
+	name = "CS Army Artilery Captain Uniform"
+	desc = "A uniform worn by an artillery commander."
+	icon_state = "Captain Artillery CS"
+	item_state = "Captain Artillery CS"
+
+/obj/item/clothing/under/urist/CSAGeneral
+	name = "CS Army General Uniform"
+	desc = "A uniform worn by an army commander."
+	icon_state = "General CS"
+	item_state = "General CS"
+
+/obj/item/clothing/under/urist/BlueCoat
+	name = "Blue Coat Infantry Uniform"
+	desc = "A simple uniform worn by infantrymen."
+	icon_state = "blue coat uniform"
+	item_state = "blue coat uniform"
+
+/obj/item/clothing/under/urist/RedCoat
+	name = "Red Coat Infantry Uniform"
+	desc = "A simple uniform worn by infantrymen."
+	icon_state = "red coat uniform"
+	item_state = "red coat uniform"
+
+/obj/item/clothing/under/urist/HamiltonClothes
+	name = "Hamilton Clothes"
+	desc = "White slacks paired with a suitable tan button up."
+	icon_state = "hamilton uniform"
+	item_state = "hamilton uniform"
+
+/obj/item/clothing/suit/urist/BlueCoatCoat
+    name = "Blue Infantry Coat"
+    desc = "A blue coat worn by soldiers."
+    icon_state = "blue coat coat"
+    item_state = "blue coat coat"
+
+/obj/item/clothing/suit/urist/RedCoatCoat
+    name = "Red Infantry Coat"
+    desc = "A red coat worn by soldiers."
+    icon_state = "red coat coat"
+    item_state = "red coat coat"
+
+/obj/item/clothing/suit/urist/HamiltonCoatCoat
+    name = "Hamilton Coat"
+    desc = "A blue coat sporting gold buttons."
+    icon_state = "hamiltoncoat"
+    item_state = "hamiltoncoat"
+
+//RS Armor Suits
+
+/obj/item/clothing/suit/urist/armor/BronzeArmor
+	name = "Bronze Armor"
+	desc = "Provides protection."
+	icon_state = "suit_bronze"
+	item_state = "suit_bronze"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS
+	w_class = 4
+	item_flags = THICKMATERIAL
+	armor = list(melee = 15, bullet = 6.6, laser = 10, energy = 2, bomb = 5, bio = 0, rad = 0)
```

Comment:
I realize these are just for adminfuckery, but shouldn't the armor values match those on the helmets?